Automattic/wp-calypso
# Plans: Add Scan Product to the plans page.

#### Changes proposed in this Pull Request

* This PR adds the basics of the scan Product to the plans page. 

After:
<img width="529" alt="Screen Shot 2020-03-25 at 1 30 54 PM" src="https://user-images.githubusercontent.com/115071/77540633-b7c3ab80-6ea3-11ea-8661-3c19006f7fde.png">
<img width="529" alt="Screen Shot 2020-03-25 at 1 26 04 PM" src="https://user-images.githubusercontent.com/115071/77540638-b98d6f00-6ea3-11ea-9143-ab3d3274b5f9.png">
<img width="521" alt="Screen Shot 2020-03-25 at 1 25 57 PM" src="https://user-images.githubusercontent.com/115071/77540640-babe9c00-6ea3-11ea-8083-bc43fae67f0c.png">

The whole solutions section now. 
<img width="1084" alt="Screen Shot 2020-03-25 at 2 25 07 PM" src="https://user-images.githubusercontent.com/115071/77541163-90211300-6ea4-11ea-9dcf-e9df850d44ca.png">s

<img width="1072" alt="Screen Shot 2020-03-25 at 2 25 36 PM" src="https://user-images.githubusercontent.com/115071/77541155-8eefe600-6ea4-11ea-9281-1c0537b23899.png">


#### Testing instructions
Load up Calyspo 
Go to http://calypso.localhost:3000/plans/monthly/example
Notice the new Scan product shows up. 

Things that don't work and will be implemented in other PRs. 
Making Checkout work. 
What the state of the card looks like when the site has that product.
